pepper-apis/docs/specification/src/components/schemas.yml

```diff
@@ -1516,8 +1516,17 @@ Validation.IntRange:
 AnnouncementModel:
   type: object
   required:
+    - guid
```

Comment:
Could you please explain why do we need `guid` here?

Reply:
I believe @bskinner suggested it. It might come in handy if frontend clients are caching the permanent messages. If so, then the guid is there so you can identify which message is which, and helps prevent displaying duplicate messages. Don't think clients need to use it if it's not needed.

Reply:
Do we have a frontend ticket for these changes? I could jump on it tomorrow.

Reply:
I don't know what's required on the frontend yet. If we're okay that permanent msg can be "dismissed" then we won't have to do anything. Otherwise, we'll need changes.

Reply:
Yep- that's it. It gives the clients a way to track individual messages (whether it's client-side only behavior, or with API interactions as well)